lib/matplotlib/pyplot.py

```diff
@@ -394,8 +393,7 @@ def draw_if_interactive() -> None:
 
     # If the manager explicitly overrides pyplot_show, use it even if a global
     # show is already present, as the latter may be here for backcompat.
-    manager_class = getattr(getattr(backend_mod, "FigureCanvas", None),
-                            "manager_class", None)
+    manager_class = getattr(backend_mod.FigureCanvas, "manager_class", None)
```

Comment:
Is this not still equal to `canvas_class` here?

Reply:
indeed